guides/common/modules/proc_removing-all-content-view-environments-from-an-activation-key.adoc

```diff
@@ -0,0 +1,22 @@
+[id="removing-all-content-view-environments-from-an-activation-key"]
+= Removing all content view environments from an activation key
+
+You can remove all content view environments from an activation key by doing either of the following:
+
+.CLI procedure
+. Remove all content view environments from the activation key by passing an empty array:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key update \
+--content-view-environment-ids [] \
+--name _My_Activation_Key_ \
+--organization-id _My_Organization_ID_
+----
+
+. Remove all content view environments from an activation key by passing an empty string:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer activation-key update --name ak1 --organization _"My_Org"_ --content-view-environments ""
```

Comment:
Can you please break this into multiple lines as the command above and improve the example values to match the pattern we use?